mozillaspeechlibrary/src/main/java/com/mozilla/speechlibrary/networking/NetworkSettings.java

```diff
@@ -0,0 +1,9 @@
+package com.mozilla.speechlibrary.networking;
+
+public class NetworkSettings {
```

Comment:
qq: why not introduce Kotlin in the codebase for handling data classes like this?